.github/workflows/publish.yml

```diff
@@ -2,10 +2,14 @@ name: publish
 
 on:
   workflow_call:
-    inputs: 
+    inputs:
       source:
         required: false
         type: string
+      verbosity:
+        required: false
+        type: string
+        default: minimal
```

Comment:
I've added this optional verbosity parameter to more easily test the NuGet build and publish for Lombiq.Privacy. You would use like this:

![image](https://user-images.githubusercontent.com/164360/163467542-48868a36-8b27-448b-8123-48921f6ea695.png)